src/openrct2-ui/windows/Ride.cpp

```diff
@@ -3349,8 +3349,9 @@ static void window_ride_mode_tweak_increase(rct_window* w)
     if (ride == nullptr)
         return;
 
-    uint8_t maxValue = RideProperties[ride->type].max_value;
-    uint8_t minValue = gCheatsFastLiftHill ? 0 : RideProperties[ride->type].min_value;
+    const auto& rideProperties = RideTypeDescriptors[ride->type].OperatingSettings;
```

Comment:
```suggestion
    const auto& operatingSettings = RideTypeDescriptors[ride->type].OperatingSettings;
```

Also applies to other occurrences. (Sorry, could have been clearer.)